PushUpdates.py

```diff
@@ -96,6 +96,13 @@
 
         instanceConfig = {}
         for line in instanceDetails:
+            if (line == ""):
+                continue
+            if (not line.__contains__(":")):
```

Comment:
It would be more pythonic to change this line to:
```suggestion
            if ":" not in line:
```
You rarely want to use magic methods (these being those methods that start and end with `__`), and the `not in` operator invokes `__contains__` under the hood anyways.